tests/coverage/RemoteServerWithPHPUnitCept.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+if (!class_exists('\PHP_CodeCoverage_Report_PHPUnit')) {
```

Comment:
It would be good to add such checks to places where this class is used and throw an exception with message that this report is only available with certain version of PHPUnit (please specify requirements).